src/utils.py

```diff
@@ -17,6 +17,7 @@
 DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
 DISCORD_CHANNEL_ID = int(os.getenv('DISCORD_CHANNEL_ID'))
 LAUNCH_TIME = os.getenv('LAUNCH_TIME')
+SKIP_REST_DAYS = bool(os.getenv('SKIP_REST_DAYS', True))
```

Comment:
Here `SKIP_REST_DAYS` will always be evaluated as `True` as [`os.getenv()`](https://docs.python.org/3/library/os.html#os.getenv) returns a string.

I think the code below is a better approach as anything that is not `true`, `1` or `t` will be evaluated as `False` :
```Python
SKIP_REST_DAYS = os.getenv('SKIP_REST_DAYS', 'TRUE').lower() in ('true', '1', 't')
```

Reply:
You're right, it might be confusing, changing that.